homeassistant/components/clickatell/notify.py

```diff
@@ -15,7 +15,11 @@
 BASE_API_URL = "https://platform.clickatell.com/messages/http/send"
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
-    {vol.Required(CONF_API_KEY): cv.string, vol.Required(CONF_RECIPIENT): cv.string}
+    {
+        vol.Required(CONF_API_KEY): cv.string,
+        vol.Required(CONF_RECIPIENT): cv.string,
+        vol.Required(CONF_SENDER): cv.string
```

Comment:
We no longer accept additions or changes to the yaml config of integrations. See: https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md

This integration needs to be refactored to use a config flow and config entries.

https://developers.home-assistant.io/docs/config_entries_config_flow_handler

Reply:
feel free to close this PR, and have buggy code. I don't want to me maintainer of the library on Pypy.